src/Service/Resolvers/MsSqlQueryBuilder.cs

```diff
@@ -75,7 +78,7 @@ public string Build(SqlInsertStructure structure)
         public string Build(SqlUpdateStructure structure)
         {
             string predicates = JoinPredicateStrings(
-                                   structure.DbPolicyPredicates,
+                                   structure.DbPolicyPredicatesForOperations[Config.Operation.Update],
```

Comment:
TryGet everywhere

Reply:
what was the resolution to this?

Reply:
created a method here:
https://github.com/Azure/data-api-builder/blob/3dc1229e6fac0d741aff66935aa24c91457e6cf7/src/Service/Resolvers/Sql%20Query%20Structures/BaseSqlQueryStructure.cs#L508